spec/jobs/email_notes_to_admins_job_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe EmailNotesToAdminsJob, type: :job do
+  def stub_scope_chain(res, *chain)
+    final = chain.pop
+    chain.each do |scope|
+      allow(Reservation).to receive(scope).and_return(Reservation)
+    end
+    allow(Reservation).to receive(final).and_return(res)
+  end
+
+  shared_examples 'admin email' do |*scopes|
+    it 'sends emails' do
+      res = spy('Array', empty?: false)
+      stub_scope_chain(res, *scopes)
+      allow(res).to receive(:update_all)
+        .with(notes_unsent: false).and_return([])
+      expect(AdminMailer).to \
+        receive_message_chain(:notes_reservation_notification, :deliver_now)
+      described_class.perform_now
+    end
+    it 'gets the appropriate reservations' do
+      expect(Reservation).to receive_message_chain(*scopes).and_return([])
```

Comment:
The return is just to skip running the job, and I'll change this to having the expectation after the setup.